proxy/src/graphql/schema.rs

```diff
@@ -150,8 +150,11 @@ impl Query {
         "1.0"
     }
 
-    fn avatar(handle: juniper::ID) -> Result<avatar::Avatar, error::Error> {
-        Ok(avatar::Avatar::from(&handle.to_string()))
+    fn avatar(handle: juniper::ID, usage: String) -> Result<avatar::Avatar, error::Error> {
+        let usage =
+            avatar::Usage::from_str(usage.as_str()).map_err(|e| error::Error::InvalidQuery(e))?;
```

Comment:
Can we move the conversion of string to `avatar::Usage` into `avatar.rs`? It would make the code here clearer.